numpy/__init__.pyi

```diff
@@ -452,7 +456,12 @@ class uint64(unsignedinteger):
     def __init__(self, __value: SupportsInt = ...) -> None: ...
 
 class inexact(number): ...  # type: ignore
-class floating(inexact, _real_generic): ...  # type: ignore
+class floating(inexact, _real_generic, float):  # type: ignore
+    # Methods which are implemented by `float` but are absent from `np.floating`
```

Comment:
```suggestion
    # Methods which are implemented by `float` but are absent from `np.floating` (xref gh-13375)
```
perhaps

Reply:
Done in https://github.com/numpy/numpy/pull/17117/commits/445b6350f0b30054f5d74d114ffca1a0348126ad.